checks/containers/container_engine/xccl_tests.py

```diff
@@ -13,28 +13,31 @@
 
 from container_engine import ContainerEngineMixin  # noqa: E402
 
-
-class XCCLTestBase(rfm.RunOnlyRegressionTest, ContainerEngineMixin):
+class XCCLTestBase(rfm.RunOnlyRegressionTest):
     valid_prog_environs = ['builtin']
     maintainers = ['amadonna', 'VCUE']
     sourcesdir = None
     test_name = parameter(['all_reduce', 'sendrecv'])
     num_nodes = variable(int, value=2)
     min_bytes = variable(str, value='1024M')
     max_bytes = variable(str, value='1024M')
-    container_env_table = {
-        'annotations.com.hooks': {
-            'aws_ofi_nccl.enabled': 'true'
-        }
-    }
+    tags = {'production', 'maintenance'}
     env_vars = {
         'NCCL_DEBUG': 'Info',
+    }
 
-        # Disable MCA components to avoid warnings
-        'PMIX_MCA_psec': '^munge',
```

Comment:
Move these env variables in the pmix hook for the ce.

Reply:
Sorry, where is that hook exactly? Do you mean inside `ContainerEngineMixin`?

Reply:
Ah, gotcha (I think), thanks.

https://github.com/eth-cscs/cscs-reframe-tests/pull/428/commits/d9e5c50acd4e0b7768ec88f1795caa4b3d529eab?